python/paddle/fluid/initializer.py

```diff
@@ -1033,6 +1034,50 @@ def _global_bias_initializer():
     return _global_bias_initializer_
 
 
+def calculate_gain(nonlinearity, param=None):
+    """
+    Get the recommended gain value of some nonlinearity function.
+
+    Args:
+        nonlinearity: nonlinearity function.
+        param: optional parameter for somme nonlinearity function
```

Comment:
param(type, optional);
default is None, means xxx

Reply:
done, thx